core/themes/seven/js/script.js

```diff
@@ -0,0 +1,324 @@
+
+            // Apply expand control's position.
+            var expandControlLeft = $lastVisibleTab.position().left + $lastVisibleTab.outerWidth();
+            $('.expand-dropdown-tabs-control', $tabsWrapper).css('left', expandControlLeft);
+            $responsiveTabsDropdown.css('right', tabArea - expandControlLeft - expandControlWidth);
+
+            // Cleanup things that may have been left over from other
+            // responsive tab strategies.
+            $primaryTabs.css('padding-left', 0);
+            expandedTabsHeaderPadding = 0;
+          }
+          else if (tabArea >= activeTabAndAfterWidth) {
+            /**
+             * 'andAfter' Responsive Tab Strategy
+             */
+            responsiveTabsType = 'andAfter';
+            accumulatedTabWidth = expandControlWidth;
+
+            // In order to get this dropdown to lay out correctly
+            // making new element that comes after the shown tabs.
+            var tabsForDropdown = [];
+            // Iterate through tabs in reverse and give appropriate classes.
+            $($tabs.get().reverse()).each(function(reverseI) {
+              var i = $tabs.length - 1 - reverseI;
+              accumulatedTabWidth += tabWidths[i];
+              if (i >= activeTabNth || accumulatedTabWidth <= tabArea) {
+                $(this).removeClass('duplicated-tab');
+              }
+              else {
+                $responsiveTabsDropdown.prepend($(this).clone());
+                $(this).addClass('duplicated-tab');
+              }
+            });
+
+            // Dropdown control gets left aligned.
+            $('.expand-dropdown-tabs-control', $tabsWrapper).css('left', 0);
+            $primaryTabs.css('padding-left', expandControlWidth);
+
+            // Manage classes on wrapper.
+            $tabsWrapper.addClass('responsive-tabs-after').removeClass('desktop-primary-tabs responsive-tabs-before responsive-tabs-mobile');
+
+            // Cleanup things that may have been left over from other
+            // responsive tab strategies.
+            expandedTabsHeaderPadding = 0;
+          }
+          else {
+            /**
+             * 'mobile' Responsive Tab Strategy.
+             */
+            responsiveTabsType = 'mobile';
+
+            // Manage classes on tabs and wrappers
+            $('.duplicated-tab', $primaryTabs).removeClass('duplicated-tab');
+            $tabsWrapper.addClass('responsive-tabs-mobile').removeClass('responsive-tabs-before responsive-tabs-after desktop-primary-tabs');
+
+            // Figure out how to lay primary tabs behind the expand control.
+            var tabsOffset = activeTabNth * (tabHeight + 2);
+            var tabsTopDistance = $tabsWrapper.position().top;
+            $primaryTabs.css('top', '-' + tabsOffset + 'px');
+            if (tabsOffset > tabsTopDistance) {
+              expandedTabsHeaderPadding = tabsOffset - tabsTopDistance + parseInt(defaultHeaderPadding, 10);
```

Comment:
I'm using to figure out padding above the menu, if the menu doesn't fit. I could just hard coded a number, but using the value that's in the CSS already seems DRYer.